test/Spec/State/Computed.spec.luau

```diff
@@ -17,7 +17,7 @@ return function()
 		local expect = getfenv().expect
 		
 		local scope = {}
-		local computed = Computed(scope, function()
+		local computed = Computed(scope, function(use, scope)
 			-- intentionally blank
```

Comment:
Use and Scope shouldn't be provided as it is never used
```suggestion
		local computed = Computed(scope, function()
```

Reply:
reverted